src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/service/StatusStore.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.service
+
+import org.springframework.stereotype.Service
+import java.util.concurrent.atomic.AtomicBoolean
+
+@Service
+class StatusStore {
```

Comment:
This is just a simple in-memory store - but if we go to multiple pods we are going to have to use the DB (and it will make testing more difficult)